deploy/blobfuse-proxy/blobfuse-proxy.yaml

```diff
@@ -47,15 +47,14 @@ spec:
             echo "Enabling blobfuse proxy systemctl service"
             systemctl daemon-reload
             systemctl enable blobfuse-proxy
-            systemctl stop blobfuse-proxy
```

Comment:
@boddumanohar FYI. This could ensure when blobfuse-proxy daemonsest restart, `blobfuse-proxy` service won't restart, then original mount would be still valid.

Reply:
Yeah. Nice edge case. Thanks for  pointing out. :)